contracts/shift.scrypt

```diff
@@ -0,0 +1,25 @@
+//
+library Shift {
+
+    // return 2^n
+    static function pow2 (int n) : int {
+
+        return unpack(

```

Comment:
💯

Reply:
What is the best way to pass arbitrary utf8 text from sdk to contract?

I've tried bytes type but only stumbled onto argument errors.